src/cloud-cli/meltano/cloud/cli/deployment.py

```diff
@@ -0,0 +1,461 @@
+
+    def prompt_for_value(self, ctx: click.Context) -> t.Any:
+        """Prompt the user to interactively select a deployment by name.
+
+        Args:
+            ctx: The Click context.
+
+        Returns:
+            The name of the deployment to be used as the default for future commands.
+        """
+        if platform.system() == "Windows":
+            asyncio.set_event_loop_policy(
+                asyncio.WindowsSelectorEventLoopPolicy(),  # type: ignore[attr-defined]
+            )
+
+        context: MeltanoCloudCLIContext = ctx.obj
+        context.deployments = asyncio.run(_get_deployments(context.config))
+        return questionary.select(
+            message="",
+            qmark="Use Meltano Cloud project deployment",
+            choices=[x["deployment_name"] for x in context.deployments],
+        ).unsafe_ask()  # Use Click's Ctrl-C handling instead of Questionary's
+
+
+@deployment_group.command("use")
+@click.option(
+    "--name",
+    "deployment_name",
+    cls=DeploymentChoicesQuestionaryOption,
+    help=(
+        "The name of a Meltano Cloud project deployment - "
+        "see `meltano cloud deployment list` for the available options."
+    ),
+    prompt=True,
+)
+@pass_context
+@run_async
+async def use_deployment(
+    context: MeltanoCloudCLIContext,
+    deployment_name: str,
+) -> None:
+    """Set a deployment as the default to use for Meltano Cloud CLI commands."""
+    deployment_name = slugify(deployment_name)
+    if context.deployments is None:  # Interactive config was not used
+        context.deployments = await _get_deployments(context.config)
+        if deployment_name not in {x["deployment_name"] for x in context.deployments}:
+            raise click.ClickException(
+                f"Unable to use deployment named {deployment_name!r} - no available "
+                "Meltano Cloud project deployment matches name.",
+            )
+    context.config.default_deployment_name = deployment_name
+    context.config.write_to_file()
+    click.secho(
+        (
+            f"Set {deployment_name!r} as the default Meltano Cloud project "
+            "deployment for future commands"
+        ),
+        fg="green",
+    )
+
+
+@deployment_group.command("create")
+@click.option(
+    "--name",
+    "deployment_name",
+    help="A name which uniquely identifies the new Meltano Cloud project deployment",
+    prompt=True,
+)
+@click.option(
+    "--environment",
+    "environment_name",
+    help="The name of the Meltano environment to be deployed",
+    prompt=True,
+)
+@click.option(
+    "--git-rev",
+    help=(
+        "The git revision this deployment should track, such as a branch name "
+        "or commit hash"
+    ),
+    prompt=True,
+)
+@pass_context
+@run_async
+async def create_deployment(
+    context: MeltanoCloudCLIContext,
+    deployment_name: str,
+    environment_name: str,
+    git_rev: str,
+) -> None:
+    """Create a new Meltano Cloud project deployment."""
```

Comment:
```suggestion
    """Create a new Meltano Cloud deployment."""
```

Reply:
@tayloramurphy Why remove the word "project"? That's what is being deployed - specifically a particular environment from a project is being deployed. Otherwise it could confuse people into thinking that they can deploy Meltano Cloud themselves, as you can with various similar open-core products.

Reply:
@WillDaSilva for consistency. Elsewhere in one of the commands we referenced a deployment without the project prefix. I'm actually okay with either (Deployment or Project Deployment) but it's likely to get shortened.